themes/default/content/docs/pulumi-cloud/esc/faq.md

```diff
@@ -0,0 +1,43 @@
+---
+title_tag: "Pulumi ESC FAQs"
+meta_desc: Frequently asked questions about preview, GA, pricing and roadmap 
+title: FAQ
+h1: Pulumi ESC FAQs
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+  pulumicloud:
+    parent: esc
+    weight: 6
+    identifier: faq
+---
+
+**Is it safe to use Pulumi ESC for my production stacks?**
+<br>
+Yes, Pulumi ESC is served using the same [highly secure](https://www.pulumi.com/security/pulumi-cloud-security-whitepaper.pdf) and highly available infrastructure as the rest of Pulumi Cloud. The service is currently in preview as we are rapidly iterating and improving the feature set. It is unlikely that we will make any breaking changes. Should we introduce a new API, we are dedicated to ensuring optimal compatibility to prevent any interruptions to your infrastructure.
```

Comment:
```suggestion
Yes, Pulumi ESC is served using the same [highly secure](/security/pulumi-cloud-security-whitepaper.pdf) and highly available infrastructure as the rest of Pulumi Cloud. The service is currently in preview as we are rapidly iterating and improving the feature set. It is unlikely that we will make any breaking changes. Should we introduce a new API, we are dedicated to ensuring optimal compatibility to prevent any interruptions to your infrastructure.
```